cmd/zetae2e/local/accounts.go

```diff
@@ -34,5 +34,5 @@ var (
 	UserAdminAddress    = ethcommon.HexToAddress("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266")
 	UserAdminPrivateKey = "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" // #nosec G101 - used for testing
 
-	FungibleAdminMnemonic = "snow grace federal cupboard arrive fancy gym lady uniform rotate exercise either leave alien grass" // #nosec G101 - used for testing
+	AdminMnemonic = "snow grace federal cupboard arrive fancy gym lady uniform rotate exercise either leave alien grass" // #nosec G101 - used for testing
```

Comment:
I think we should keep the name `FungibleAdmin`. The same identifier is used in other reports, such as protocol contracts as well . Having the same name makes reading and debugging code easier when working across repos.

With that said however , I do agree with using just Admin , but maybe we should do it in a separate PR , nad rename the functions on the smart contracts as well

Reply:
Ok

Reply:
58b2d923